tests/framework/extensions/provisioning/creates.go

```diff
@@ -49,6 +49,13 @@ func CreateProvisioningCluster(client *rancher.Client, provider Provider, cluste
 		return nil, err
```

Comment:
It seems like this file is missing `provisioninginput` package import, which creates a single line diff from 2.8

Reply:
Addressed